src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
```

Comment:
Instead of using a magic number '3' directly in the loop, define it as a named constant at the beginning of your class to improve code readability and maintainability.